hazelcast/src/test/java/com/hazelcast/map/impl/query/QueryBasicTest.java

```diff
@@ -401,19 +401,19 @@ private void testPredicateDateAttribute(IMap map) {
     @Test(timeout = 1000 * 60)
     public void testPredicateEnumAttribute() {
         HazelcastInstance instance = createHazelcastInstance(getConfig());
-        IMap map = instance.getMap("testPredicateEnumAttribute");
+        IMap<Integer, NodeType> map = instance.getMap("testPredicateEnumAttribute");
         testPredicateEnumAttribute(map);
     }
 
     @Test(timeout = 1000 * 60)
     public void testPredicateEnumAttributeWithIndex() {
         HazelcastInstance instance = createHazelcastInstance(getConfig());
-        IMap map = instance.getMap("testPredicateEnumAttribute");
+        IMap<Integer, NodeType> map = instance.getMap("testPredicateEnumAttribute");
         map.addIndex("this", true);
-        testPredicateDateAttribute(map);
+        testPredicateEnumAttribute(map);
```

Comment:
Here is the fixed method call: `testPredicateDateAttribute()` vs. `testPredicateEnumAttribute()`.